cli_tools/gce_vm_image_import/importer/args.go

```diff
@@ -52,6 +53,7 @@ type ImportArguments struct {
 	Family               string
 	GcsLogsDisabled      bool
 	ImageName            string
+	Inspect              bool
```

Comment:
Is this the flag from PR description that is referred to as -import? Probably a typo in the description?

Reply:
thanks! good find. updated the description